crates/neon-runtime/src/napi/date.rs

```diff
@@ -0,0 +1,17 @@
+use std::mem::MaybeUninit;
+use crate::napi::bindings as napi;
+use crate::raw::{Env, Local};
+
+pub unsafe fn new_date(env: Env, value: f64) -> Local {
```

Comment:
Please add doc comments with a `# Safety` section.

Reply:
I don't think I understand how this API is unsafe. I see that this is using the `unsafe` keyword but I'm still not really sure what this code is doing.

Reply:
`Env` is a raw pointer. It's the caller's responsibility to ensure it points to a `napi_env` that is valid for the current context.

Reply:
Ah okay that makes sense, thanks! 😃